tools/loader/ImageLoader.cpp

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
Did you make sure to use 'git move' to preserve the file history?

Reply:
I did for `loader.cpp` to `Loader.cpp`.  So doing `git log --follow ./tools/loader/Loader.cpp` shows the history for `loader.cpp` + `Loader.cpp`.

However `Loader.h` and `ImageLoader.cpp` have no history... Is it possible/desired to basically replicate the history for `loader.cpp` into those files too, so all three have common history up until this commit?